docs/book/src/clusterctl/configuration.md

```diff
@@ -7,11 +7,11 @@ The `clusterctl` config file is located at `$HOME/.cluster-api/clusterctl.yaml`
 
 ## Provider repositories
 
-The `clusterctl` CLI is designed to work with providers implementing the [clusterct Provider Contract](contract.md).
+The `clusterctl` CLI is designed to work with providers implementing the [clusterctl Provider Contract](provider-contract.md).
 
 Each provider is expected to define a provider repository, a well-known place where release assets are published. 
 
-By default, `clusterctl` ships with providers sponsored by SIG Cluster Lifecycle. 
+By default, `clusterctl` ships with `aws`, `vsphere` and `docker` providers sponsored by SIG Cluster Lifecycle.
```

Comment:
I'm not sure we should to hard code the list here, because then we have to maintain it consistent with the code.

Reply:
Hmm...that's a good point but when I read that 
> `clusterctl`ships with providers sponsored by SIG Cluster Lifecycle

I was immediately curious as to which ones. We can maybe state that these default providers are subject to change in the future.
WDYT?

Reply:
what about adding "You can use the `clusterctl config providers` command to get the list of supported providers."

Reply:
Clearly I need to play around with `clusterctl` some more 😄

Reply:
it is a moving target ....  🤣